README.md

```diff
@@ -301,13 +302,15 @@ It is possible for an HTML document to contain multiple elements with `popup=pop
 
 ## Accessibility / Semantics
 
-Since the `toplayer` content attribute can be applied to any element, and only impacts the element’s presentation (top layer vs not top layer), this should not have **any semantic or accessibility impact**. I.e. the element with the `toplayer` attribute will keep its existing semantics and AOM representation.
+Since the `toplayer` content attribute can be applied to any element, and only impacts the element’s presentation (top layer vs not top layer), this should not have **any semantic or accessibility impact**. I.e., the element with the `toplayer` attribute will keep its existing semantics and AOM representation. In the event an author needs to extend or modify a particular element's ARIA semantics, this may be done in accordance to existing allowances of [ARIA in HTML](https://w3c.github.io/html-aria/).
```

Comment:
is `toplayer` still used, or should this be replaced with `popup`?

Reply:
Thanks for the PR. And oops - yes that should be `popup` and not `toplayer`. I can fix that. I'll accept this PR though.